test/trento_web/controllers/v1/suma_credentials_controller_test.exs

```diff
@@ -38,6 +38,80 @@ defmodule TrentoWeb.V1.SUMACredentialsControllerTest do
     end
   end
 
+  describe "create new user settings" do
+    test "should save new valid settings if no previous settings have been saved", %{conn: conn} do
+      settings =
+        %{url: url, username: username} = %{
+          url: Faker.Internet.image_url(),
+          username: Faker.Internet.user_name(),
+          password: Faker.Lorem.word(),
+          ca_cert: Faker.Lorem.sentence()
+        }
+
+      %{"ca_uploaded_at" => ca_uploaded_at} =
+        resp =
+        conn
+        |> put_req_header("content-type", "application/json")
+        |> post("/api/v1/settings/suma_credentials", settings)
+        |> json_response(:created)
+
+      assert %{"url" => ^url, "username" => ^username} = resp
+      assert ca_uploaded_at != nil
+    end
+
+    test "should not save valid settings when previously settings have been saved", %{conn: conn} do
+      insert(:software_updates_settings, [ca_cert: nil, ca_uploaded_at: nil],
```

Comment:
just `insert(:software_updates_settings, [ca_cert: nil, ca_uploaded_at: nil])`.
The on conflict options are not needed anymore.